Makefile

```diff
@@ -37,7 +38,7 @@ endif
 
 all: yaml build image
 
-build: chaosdaemon manager chaosfs dashboard dashboard-server-frontend
+build: dashboard-server-frontend
```

Comment:
miss binary here?

Reply:
No. Building binary on host is useless. All binary will be built in image.

For debug use (such as check grammer...), you can `make binary` manually.

Reply:
emm, I thought our CI run `make build`, but I'm wrong. That's ok.